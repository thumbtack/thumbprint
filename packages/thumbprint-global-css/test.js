const prettier = require('prettier');
const fs = require('fs');

it('compiles correctly', () => {
    const css = fs.readFileSync(
        'packages/thumbprint-global-css/dist/thumbprint-global.css',
        'utf-8',
    );
    // Format the CSS to make it easier to read the diff.
    const formattedCSS = prettier.format(css, {
        parser: 'css',
    });

    expect(formattedCSS).toMatchSnapshot();
});
