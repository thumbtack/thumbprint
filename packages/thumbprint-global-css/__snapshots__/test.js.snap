// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
"html {
  box-sizing: border-box;
}
*,
:after,
:before {
  box-sizing: inherit;
}
body {
  margin: 0;
}
blockquote,
caption,
dd,
dl,
fieldset,
figure,
form,
h1,
h2,
h3,
h4,
h5,
h6,
hr,
legend,
ol,
p,
pre,
table,
td,
th,
ul {
  margin: 0;
  padding: 0;
}
ol,
ul {
  list-style: none;
}
h1,
h2,
h3,
h4,
h5,
h6 {
  font-weight: 400;
  font-size: inherit;
}
table {
  border-collapse: collapse;
  border-spacing: 0;
}
abbr[title],
dfn[title] {
  cursor: help;
}
ins,
u {
  text-decoration: none;
}
ins {
  border-bottom: 1px solid;
}
img {
  font-style: italic;
}
button,
input[type=\\"checkbox\\"] + label,
input[type=\\"radio\\"] + label,
option,
select {
  cursor: pointer;
}
[type=\\"email\\"]:active,
[type=\\"email\\"]:focus,
[type=\\"number\\"]:active,
[type=\\"number\\"]:focus,
[type=\\"password\\"]:active,
[type=\\"password\\"]:focus,
[type=\\"search\\"]:active,
[type=\\"search\\"]:focus,
[type=\\"text\\"]:active,
[type=\\"text\\"]:focus,
[type=\\"url\\"]:active,
[type=\\"url\\"]:focus,
textarea:active,
textarea:focus {
  cursor: text;
  outline: none;
}
[type=\\"email\\"],
[type=\\"number\\"],
[type=\\"password\\"],
[type=\\"search\\"],
[type=\\"text\\"],
[type=\\"url\\"],
textarea {
  -webkit-appearance: none;
}
a {
  text-decoration: none;
}
a:active,
a:hover {
  outline: 0;
}
sub,
sup {
  font-size: 100%;
  vertical-align: baseline;
}
em {
  font-style: italic;
}
b,
strong {
  font-weight: 700;
}
fieldset {
  border: 0;
}
textarea {
  vertical-align: top;
}
button[disabled],
html input[disabled],
select[disabled],
textarea[disabled] {
  cursor: default;
}
button,
select {
  text-transform: none;
}
button,
input,
select,
textarea {
  font-family: inherit;
  font-size: 100%;
  margin: 0;
}
button::-moz-focus-inner {
  padding: 0;
  border: 0;
}
input[type=\\"search\\"]::-webkit-search-cancel-button,
input[type=\\"search\\"]::-webkit-search-decoration {
  -webkit-appearance: none;
}
button,
html input[type=\\"button\\"],
input[type=\\"reset\\"],
input[type=\\"submit\\"] {
  -webkit-appearance: button;
  cursor: pointer;
}
[required] {
  box-shadow: none;
}
body {
  color: #2f3033;
  font-family: Mark, Avenir, Helvetica, Arial, sans-serif;
  font-size: 16px;
  line-height: 1.6;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}
a {
  fill: #009fd9;
  cursor: pointer;
}
a,
a:active,
a:hover {
  color: #009fd9;
}
/*# sourceMappingURL=thumbprint-global.css.map */
"
`;
